frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1018,29 +1019,144 @@ function BookMapWidget:update()
 end
 
 
-function BookMapWidget:showHelp()
+function BookMapWidget:showMenu()
+    local button_dialog
+    -- Width of our -/+ buttons, so it looks fine with Button's default font size of 20
+    local plus_minus_width = Screen:scaleBySize(60)
+    local buttons = {
+        {{
+            text = _("About book map"),
+            align = "left",
+            callback = function()
+                self:showAbout()
+            end,
+        }},
+        {{
+            text = _("Available gestures"),
+            align = "left",
+            callback = function()
+                self:showGestures()
+            end,
+        }},
+        {{
+            text = _("Switch current/initial views"),
+            align = "left",
+            enabled_func = function() return self.toc_depth > 0 end,
+            callback = function()
+                self:toggleDefaultSettings()
+            end,
+        }},
+        {{
+            text = _("Switch grid/flat views"),
+            align = "left",
+            enabled_func = function() return self.toc_depth > 0 end,
+            callback = function()
+                self.flat_map = not self.flat_map
+                self:saveSettings()
+                self:update()
+            end,
+        }},
+        {
+            {
+                text = _("Chapters"),
+                callback = function() end,
+                align = "left",
+            },
+            {
+                text = "\u{2796}", -- Heavy minus sign
+                enabled_func = function() return self.toc_depth > 0 end,
+                callback = function()
+                    if self:updateTocDepth(self.flat_map and 1 or -1, nil) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            },
+            {
+                text = "\u{2795}", -- Heavy plus sign
+                enabled_func = function() return self.toc_depth < self.max_toc_depth end,
+                callback = function()
+                    if self:updateTocDepth(self.flat_map and -1 or 1, nil) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            }
+        },
+        {
+            {
+                text = _("Page slots width"),
```

Comment:
Page slot widths? I'm not completely sure what this is for.

Reply:
It is mentionned (was and still is, didn't change its wording) in the help text:
![image](https://user-images.githubusercontent.com/24273478/226935395-74653c48-21d4-45bc-a09a-0401374f0599.png)

Its for going from less or more wide page slots:
![image](https://user-images.githubusercontent.com/24273478/226936413-90e5868d-b51d-4bd3-b24d-ae091c3f93cf.png)
![image](https://user-images.githubusercontent.com/24273478/226936323-495877a3-1225-41a3-8ce5-4fc10d67e6d1.png)
![image](https://user-images.githubusercontent.com/24273478/226936509-0248f759-1d2b-4566-a817-fadd547c2756.png)

Reply:
Then just make it page slot width. :)

Reply:
OK... always confused with these kind of English constructs....
Are you ok with my "Thunbnail rows" and "Thumbnail columns" with Thumbnail singular ? (meaning, number of thumbnail rows, how many rows of thumbnails...)

Reply:
Yes, it's the opposite that'd be weird. ^_^ "Page slot widths" would be fine too but I don't think that's intended here?